scripts/utils/constructProposalCalldata.ts

```diff
@@ -91,5 +102,25 @@ function getTimelockCalldata(proposal: ExtendedAlphaProposal, proposalInfo: Temp
     salt
   ]);
 
-  return `Calldata: ${calldata}\nExecute Calldata: ${executeCalldata}`;
+  if (podConfig) {
+    const proposalId = calcProposalId(proposal.targets, proposal.values, combinedCalldatas, predecessor, salt);
+    const registerMetadataCalldata = metadataFuncFrag.encodeFunctionData('registerProposal', [
+      podConfig.id,
+      proposalId,
+      proposal.description
+    ]);
+    return `Calldata: ${calldata}\nMetadata Calldata: ${registerMetadataCalldata}\nExecute Calldata: ${executeCalldata}`;
```

Comment:
It's strange to me that we're returning a formatted string. If we want to log this to console, we should do that explicitly; the string returned should just be the pure calldata, imo.

Reply:
Yep, that's what I'm saying - the logging should happen at the higher level